src/QsCompiler/Transformations/ClassicallyControlledTransformation.cs

```diff
@@ -226,41 +315,15 @@ private TypedExpression CreateApplyIfExpression(QsResult result, TypedExpression
                         return null; // ToDo: Diagnostic message - cond body not valid
```

Comment:
```suggestion
                        return null; // ToDo: Diagnostic message - conditional block not valid
```
To be consistent with the naming in the SyntaxTree.